common/script/libs/achievements.js

```diff
@@ -0,0 +1,507 @@
+import i18n from '../i18n';
+
+let achievs = {};
+let index;
+
```

Comment:
Adding the achievements to the content api is a good choice, but now the achievements result just has some string-keys not the actual values (like every other property in the content), I guess they should have the localized values right? @crookedneighbor 

@khipkin good job already extracting those keys :+1: 

Any 3rd party application would still rather use the /members/{mid}/achievements api call to get the achievements, instead of enumerating through the content array and trying to recreate the achievements client-wise.

@khipkin gonna test the changes tomorrow, but on the first look it probably still works as before^^